packages/lit-html/src/lit-html.ts

```diff
@@ -1439,6 +1454,33 @@ export class EventPart extends AttributePart {
   }
 }
 
+export class ElementPart {
+  readonly type = ELEMENT_PART;
+  /** @internal */
+  _directive?: Directive;
+  // This is to ensure that every Part has a _value.
+  _$value: undefined;
+
+  /** @internal */
+  _$parent: Disconnectable | undefined;
+  /** @internal */
+  _$disconnetableChildren?: Set<Disconnectable> = undefined;
+  /** @internal */
+  _setDirectiveConnected?: (
+    directive: Directive | undefined,
+    isConnected: boolean,
+    removeFromParent?: boolean
+  ) => void = undefined;
+
+  constructor(public element: Element, parent: Disconnectable) {
```

Comment:
Can the actual code here be in the `Part` base class to be more DRY?

Reply:
I have a stash where I tried making `ElementPart` a base class to `AttributePart` (since it's basically a superset of it), but it still needs a separate class, `_$setValue` implementation, and `AttributePart` then needs to super to it, so the checksize didn't change meaningfully (to the number of digits it shows).

Reply:
ok